app/controllers/api/v1/comments_controller.rb

```diff
@@ -1,8 +1,15 @@
 class Api::V1::CommentsController < Api::V1::BaseController
   
+  before_filter :set_user_id, only: :create
+  load_and_authorize_resource
+  
+  def set_user_id
+    params[:user_id] = current_user.id
```

Comment:
private function. also could throw it into base controller so that you don't create it twice.